langchain/src/agents/chat/index.ts

```diff
@@ -165,7 +165,7 @@ export class ChatAgent extends Agent {
       args?.outputParser ?? ChatAgent.getDefaultOutputParser();
 
     return new ChatAgent({
-      llmChain: chain,
+      runnable: chain,
```

Comment:
I don't think we need to change old agents, just update what `AgentExecutor` accepts?

https://python.langchain.com/docs/modules/agents/how_to/agent_structured#create-the-agent